cairo/tests/ethereum/cancun/vm/instructions/test_environment.py

```diff
@@ -73,8 +80,15 @@
     .build()
 )
 
-code_access_size_strategy = st.integers(min_value=0, max_value=MAX_CODE_SIZE).map(U256)
-code_start_index_strategy = code_access_size_strategy
+
+def create_bounded_u256_strategy(min_value: int = 0, max_value: int = MAX_CODE_SIZE):
+    return st.integers(min_value=min_value, max_value=max_value).map(U256)
```

Comment:
the default value for a strategy named "bounded_u256" should not be MAX_CODE_SIZE but `2**256 - 1`. very confusing otherwise as you'd expect the default behavior to return the full range of numbers in a u256

Reply:
also unless you 're using these strategies in multiple places, I would inline them in the only place where they're used (for example calldatacopy_strategy)

Reply:
and now that there's a function to dynamically create bounded U256 strategies, I would not even create intermediate variables but directly call them like 

`    data_start_index = draw(bounded_u256(max_value=MAX_CODE_SIZE))`

Reply:
good point, on it

Reply:
Done with [daf290c](https://github.com/kkrt-labs/keth/pull/504/commits/daf290ca7c69818da4eb011eb4461285bd786349)